scripts/update-patches.sh

```diff
@@ -13,13 +13,13 @@ GLUONDIR="$(pwd)"
 for module in $GLUON_MODULES; do
 	echo "--- Updating patches for module '$module' ---"
 
-	rm -rf "${GLUON_PATCHESDIR}/$module"
+	rm -rf "${GLUON_PATCHESDIR:?}/$module"
```

Comment:
Can you explain this?

Reply:
if ${GLUON_PATCHESDIR} and $module get emty at same time like:
${GLUON_PATCHESDIR} = ""
$module = ""

rm -rf "${GLUON_PATCHESDIR}/$module" will turn into rm -rf "/"